modules/apps/business-productivity/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/css/main.scss

```diff
@@ -1,5 +1,6 @@
 @import "bourbon";
 @import "mixins";
+@import "form-builder-field-toolbar";
```

Comment:
@brunobasto @fernandosouza On this one, why are we importing twice, once without namespacing and once with namespacing (on line 633)?
Also, should the file have it's own CSS file, or should that file be a Sass partial?